vitals/Main.java

```diff
@@ -1,20 +1,35 @@
 package vitals;
 
-public class Main {
-    static boolean vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
-        }
-        return true;
-    }
+import vitals.Vitalfun;
 
+public class Main {
+   
     public static void main(String[] args) {
-        assert(vitalsAreOk(80, 97, 40) == true);
-        assert(vitalsAreOk(60, 98, 40) == false);
-        System.out.println("Some more tests needed");
+        
+         Vitalfun x = new Vitalfun();
+         x.setAlert(new AlertbySound());
+        //These are cases to test checkvitallimit function
+        
+        /*assert(checkVitalLimit(66, 75, 100)) == false;
```

Comment:
Do not leave commented code in commits. If we need the old code, it will be there in previous commits.

Reply:
Thanks for the feedback Sudeep, I modified it as you suggested